pkg/monitors/filesystems/filesystems.go

```diff
@@ -66,6 +71,17 @@ func (m *Monitor) getCommonDimensions(partition *gopsutil.PartitionStat) map[str
 		"device":     strings.Replace(partition.Device, " ", "_", -1),
 		"fs_type":    strings.Replace(partition.Fstype, " ", "_", -1),
 	}
+	if m.sendModeDimension {
+		var mode string
+		if partition.Opts != "" {
+			mode = partition.Opts[:2]
```

Comment:
This seems really sketchy, are you sure it's always the first two chars of the options?

Reply:
I agree it's sketchy, but yeah after diving into how the field is set, it is always the first two characters.
I don't see another way to get the information other than this string.

Examples:
https://github.com/shirou/gopsutil/blob/master/disk/disk_darwin.go#L29-L32
https://github.com/shirou/gopsutil/blob/master/disk/disk_linux.go#L272-L284
https://github.com/shirou/gopsutil/blob/master/disk/disk_windows.go#L121-L127

Reply:
Given that options are always comma separated (I'm at least more confident of that than the fact that the mode option always comes first), just do a strings.Split(opts, ",") and see if one of the items is `ro` or `rw`.  That seems at least a bit more robust.

Reply:
That was the other way I was considering. Sounds good to me.